### ISSUE TITLE
Referencing unbound variable in certbot.display.ops.get_email

### ISSUE THREAD

**Issue by ieugen** [certbot/certbot#3672]
Hello, 

I have a new DNS domain. I have added an entry in the nameserver but it seems not to be propagated. 

When I ran certbot certonly I got this error and stacktrace:

````
2016-10-21 14:33:08,109:DEBUG:requests.packages.urllib3.connectionpool:"POST /acme/new-reg HTTP/1.1" 400 94
2016-10-21 14:33:08,111:DEBUG:root:Received <Response [400]>. Headers: {'content-length': '94', 'boulder-request-id': 'FVDE0vBeNybRDlP0D8_wS7ChtpBoriwlYviCgM-f7Ow', 'expires': 'Fri, 21 Oct 2016 14:33:08 GMT', 'server': 'nginx', 'connection': 'close', 'pragma': 'no-cache', 'cache-control': 'max-age=0, no-cache, no-store', 'date': 'Fri, 21 Oct 2016 14:33:08 GMT', 'content-type': 'application/problem+json', 'replay-nonce': 'p1RG6Y8H_CKk4IdUmpEyNKxIv5yMvQ7HwQOlZvyC5t4'}. Content: '{\n  "type": "urn:acme:error:invalidEmail",\n  "detail": "empty DNS response",\n  "status": 400\n}'
2016-10-21 14:33:08,111:DEBUG:acme.client:Storing nonce: '\xa7TF\xe9\x8f\x07\xfc"\xa4\xe0\x87T\x9a\x9124\xacH\xbf\x9c\x8c\xbd\x0e\xc7\xc1\x03\xa5f\xfc\x82\xe6\xde'
2016-10-21 14:33:08,111:DEBUG:acme.client:Received response <Response [400]> (headers: {'content-length': '94', 'boulder-request-id': 'FVDE0vBeNybRDlP0D8_wS7ChtpBoriwlYviCgM-f7Ow', 'expires': 'Fri, 21 Oct 2016 14:33:08 GMT', 'server': 'nginx', 'connection': 'close', 'pragma': 'no-cache', 'cache-control': 'max-age=0, no-cache, no-store', 'date': 'Fri, 21 Oct 2016 14:33:08 GMT', 'content-type': 'application/problem+json', 'replay-nonce': 'p1RG6Y8H_CKk4IdUmpEyNKxIv5yMvQ7HwQOlZvyC5t4'}): '{\n  "type": "urn:acme:error:invalidEmail",\n  "detail": "empty DNS response",\n  "status": 400\n}'
2016-10-21 14:33:10,918:WARNING:certbot.util:Invalid email address: gr8pi.org.
2016-10-21 14:33:10,922:DEBUG:certbot.main:Exiting abnormally:
Traceback (most recent call last):
  File "/usr/bin/certbot", line 9, in <module>
    load_entry_point('certbot==0.8.1', 'console_scripts', 'certbot')()
  File "/usr/lib/python2.7/dist-packages/certbot/main.py", line 744, in main
    return config.func(config, plugins)
  File "/usr/lib/python2.7/dist-packages/certbot/main.py", line 550, in obtain_cert
    le_client = _init_le_client(config, auth, installer)
  File "/usr/lib/python2.7/dist-packages/certbot/main.py", line 360, in _init_le_client
    acc, acme = _determine_account(config)
  File "/usr/lib/python2.7/dist-packages/certbot/main.py", line 345, in _determine_account
    config, account_storage, tos_cb=_tos_cb)
  File "/usr/lib/python2.7/dist-packages/certbot/client.py", line 120, in register
    regr = perform_registration(acme, config)
  File "/usr/lib/python2.7/dist-packages/certbot/client.py", line 153, in perform_registration
    config.namespace.email = display_ops.get_email(invalid=True)
  File "/usr/lib/python2.7/dist-packages/certbot/display/ops.py", line 63, in get_email
    elif suggest_unsafe:
UnboundLocalError: local variable 'suggest_unsafe' referenced before assignment
````

**Comment by bmw** [comment]
@ieugen, thanks for reporting this.

While this is a bug that we will fix in our next release, what's going on here is the e-mail address you are providing to Certbot is invalid. If you fix this, it will work around the problem.

**Comment by bmw** [comment]
Fixed in #3685.
